types/react-dropzone/v2/react-dropzone-tests.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import Dropzone = require('react-dropzone');
+
+class Test extends React.Component {
+  constructor(props: any) {
```

Comment:
this appears to be out of sync w/ v1. (updates were made there that are not here)